CHANGELOG.md

```diff
@@ -2,11 +2,24 @@
 
 ## v6.0.0-beta.4 - 2017-07-12
 
-- Add Windows PowerShell `PSModulePath` by default on Windows. (#4132)
-- Move PowerShell to `2.0.0-preview3-25426-01` and using the .NET CLI version `2.0.0-preview2-006502`. (#4144)
+## Windows PowerShell backwards compatibility
+
+In the `beta.4` release, we've introduced a change to add the Windows PowerShell `PSModulePath` to the default $PSModulePath in PowerShell Core on Windows. (#4132)
```

Comment:
whoops, meant it to just be `PSModulePath` (the `$env:` is too confusing). Changing now.